guides/source/caching_with_rails.md

```diff
@@ -512,6 +512,26 @@ class ProductsController < ApplicationController
 end
 ```
 
+Sometimes we want to cache response (static page) that never gets expired.
+To achieve that, we can use `http_cache_forever` helper and by doing so browser
+and proxies will cache it indefinitely.
+
+By default cached response will be private. To make it public response, pass
+`public: true` parameter to this method. In this helper, `last_modified` header
+is set to `Time.new(2011, 1, 1).utc`. Please use this method carefully as
```

Comment:
Move the last line to a warning:

WARNING: Use this method carefully as browser/proxy won't able to invalidate the cached response unless browser cache is forcefully cleared.